index.html

```diff
@@ -562,7 +563,75 @@ <h2>Dictionary {{RTCRtpEncodingParameters}} Members</h2>
         allows the <a>user agent</a> to adapt its bandwidth allocation strategy
         based on the current network conditions.</p>
       </dd>
+      <dt><dfn data-idl>codec</dfn> of type <span class="idlMemberType">RTCRtpCodec</span></dt>
+      <dd>
+        <p>Indicates the codec for this encoding. The {{RTCRtpCodec}} dictionary is
```

Comment:
We need to clarify when to use or not use this codec.

For backwards-compat, someone using get/setParameters() should just get "whatever was negotiated", i.e. the order in the SDP.

But if someone says "I want to control codec using this API", I think we need to say to ignore the SDP order, otherwise you would reset the codec in the next SDP negotiation for unrelated reasons.

I suggest the opt-in to control the codec with this API instead of setCodecPreferences() should be "if codec was specified in addTransceiver()". Boolean internal slot for this?

Reply:
If codec is a preference, undefined by default, that solves it too

Reply:
How about codecPreferences: undefined by default, once set it overrides the preference order - let's be clear about that